os/android/platformdefines.h

```diff
@@ -51,6 +49,33 @@ struct gralloc_handle {
 };
 
 typedef struct gralloc_handle* HWCNativeHandle;
+typedef native_handle_t HWCNativeBuffer;
+#define GETNATIVEBUFFER(handle) (*(handle->handle_))
+
+struct BufferHash {
+  size_t operator()(native_handle_t const& p) const {
+    std::size_t seed = 0;
+    hash_combine_hwc(seed, p.numFds);
+    for (int i = 0; i < p.numFds; i++) {
+      hash_combine_hwc(seed, (const size_t)p.data[i]);
```

Comment:
shouldn't we have width, height, format etc like linux ?

Reply:
on Android, the only input is buffer_handle_t
See
HWC2::Error IAHWC2::Hwc2Layer::SetLayerBuffer(buffer_handle_t buffer,
                                              int32_t acquire_fence) 

numFds and data are the only members of buffer_handle_t